Build failure with latest go 1.12.6

I am trying to build the project following instructions using latest go `1.12.6` but i get the following error:

```
user@host:~# go get -u github.com/cycloidio/terracognita 
# github.com/cycloidio/terracognita/aws
programmi/go/src/github.com/cycloidio/terracognita/aws/provider.go:46:32: impossible type assertion:
	*"github.com/hashicorp/terraform/helper/schema".Provider does not implement "github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/terraform".ResourceProvider (wrong type for Apply method)
		have Apply(*"github.com/hashicorp/terraform/terraform".InstanceInfo, *"github.com/hashicorp/terraform/terraform".InstanceState, *"github.com/hashicorp/terraform/terraform".InstanceDiff) (*"github.com/hashicorp/terraform/terraform".InstanceState, error)
		want Apply(*"github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/terraform".InstanceInfo, *"github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/terraform".InstanceState, *"github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/terraform".InstanceDiff) (*"github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/terraform".InstanceState, error)
```

--- a/Makefile
+++ b/Makefile
@@ -62,6 +62,10 @@ dbuild: ## Builds the docker image with same name as the binary
 build: ## Builds the binary
 	GO111MODULE=on CGO_ENABLED=0 GOARCH=amd64 go build -o $(BIN) ${LDFLAGS}
 
+.PHONY: install
+install: ## Install the binary
+	GO111MODULE=on CGO_ENABLED=0 GOARCH=amd64 go install ${LDFLAGS}
+
 .PHONY: clean
 clean: ## Removes binary and/or docker image
 	rm -f $(BIN)

--- a/README.md
+++ b/README.md
@@ -13,14 +13,20 @@ We decided to opensource this tool as we believe that it will help people to ado
 
 ## Installation
 
-It uses Go Modules, so GO 11 or higher is required.
+It uses Go Modules, so GO 1.11+ is required.
 If you wish to run it via docker then `docker` binary is required.
 While if you wish to run it locally; you will need to have the extra `brz` lib installed.
 
+You have 2 options to get the package:
+
+* Clone the repository to `$GOPATH/src/github.com/cycloidio/terracognita`
+* `go get -d github.com/cycloidio/terracognita`
+
+Then feel free to play with it :)
+
 ```
-$> go get -u github.com/cycloidio/terracognita
 $> cd $GOPATH/src/github/cycloidio/terracognita
-$> go install .
+$> make install
 ```
 
 ## Versions

